pkg/users/accounts-list.js

```diff
@@ -50,10 +51,11 @@ const _ = cockpit.gettext;
 const GroupActions = ({ group, accounts }) => {
     const [isKebabOpen, setKebabOpen] = useState(false);
 
-    if (!superuser.allowed)
-        return null;
-
     const actions = [
+        <DropdownItem key="edit-group"
+                      onClick={ev => { ev.preventDefault(); cockpit.location.go(["group", group.name]) }}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230110-115950-e62ee64c-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)